_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -21677,9 +21677,6 @@
 /obj/effect/spawner/window/reinforced,
 /turf/simulated/floor/plating,
 /area/hallway/primary/port)
-"aYS" = (
-/turf/simulated/wall/r_wall,
-/area/escapepodbay)
```

Comment:
Kill the area in the dm files while youre at it.

Reply:
2l84u